### PR TITLE
chore(components): use the post-collapsible-trigger within the post-accordion-item

### PR DESCRIPTION
**Pull request by alizedebray**

### CODE REVIEW

File: packages/components/src/components/post-collapsible-trigger/post-collapsible-trigger.scss
Comment (gfellerph) [inline packages/components/src/components/post-collapsible-trigger/post-collapsible-trigger.scss:1]: I don't think this is necessary, buttons are not block level elements by default. I'd just not ship any styles at all.

File: .changeset/plenty-experts-admire.md
Comment (alizedebray) [inline .changeset/plenty-experts-admire.md:1]: Remove file

File: .changeset/thin-eyes-join.md
Comment (alizedebray) [inline .changeset/thin-eyes-join.md:1]: Remove file

**Review by oliverschuerch — COMMENTED** [review]
@alizedebray did you do something within this PR, which is was not implemented in the v7 patch and the cherry-picked PR for v8?
Because if the answer is "YES", you should add a changeset to this PR ;)

File: packages/components/src/components/post-collapsible-trigger/post-collapsible-trigger.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import { Component, Element, Listen, Method, Prop, Watch } from '@stencil/core';
-import { version } from '@root/package.json';
-import { checkNonEmpty, checkType, debounce } from '@/utils';
+import { Component, Element, Method, Prop, Watch } from '@stencil/core';
+import { version } from 'typescript';
```
Comment (oliverschuerch) [inline packages/components/src/components/post-collapsible-trigger/post-collapsible-trigger.tsx:2]: Nope!
```suggestion
import { version } from '@root/package.json';
```

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

**Review by oliverschuerch — APPROVED** [review]

**Pull request merged**